library/src/jsMain/kotlin/dev/bluefalcon/BlueFalcon.kt

```diff
@@ -53,7 +53,7 @@ actual class BlueFalcon actual constructor(context: ApplicationContext, serviceU
             .then { bluetoothDevice ->
                 val device = BluetoothPeripheral(bluetoothDevice)
                 delegates.forEach {
-                    it.didDiscoverDevice(device)
+                    it.didDiscoverDevice(device, mapOf()) //TODO
```

Comment:
should we put the default value in the definition?

Reply:
I don't know exactly what you mean, but I added something. Is that what you meant?